pkg/didcomm/dispatcher/outbound.go

```diff
@@ -234,3 +232,13 @@ func (o *OutboundDispatcher) addTransportRouteOptions(req []byte, des *service.D
 
 	return req, nil
 }
+
+// TODO - inject MediaType selection strategy.
```

Comment:
I guess the packager should be deducting the mediaType based on its list of packers, this would be done in Packager.PackMessage() above, in this case, we should just return the full list of MediaTypes to the packager, what do you think @llorllale ?

Reply:
@Baha-sk i wrote that TODO on the assumption that the user would declare their preferred media type when responding (`accept` is a list).

Personally i would leave the packager to choose the packer and the user to declare the media type they want to use.

Reply:
correct, this is why it would make sense to return the full list of MediaTypes here, it can be done in a followup pr